test/skimNtuple_HHbtag.cpp

```diff
@@ -5185,125 +5272,132 @@ int main (int argc, char** argv)
 		  // Boosted section
 		  theSmallTree.m_isBoosted = 0;
 		  if (theBigTree.ak8jets_px->size() > 0)
+		    {  
+		      vector<pair<float, int>> fatjets_pT;
+		      vector<pair<float, int>> fatjets_bTag;
+		      for (unsigned int ifj = 0; ifj < theBigTree.ak8jets_px->size(); ++ifj)
 			{
-			  // int idxSub1 = -1;
-			  // int idxSub2 = -1;
-			  // int idxFatj = -1;
-
-			  vector<pair<float, int>> fatjets_bTag;
-			  for (unsigned int ifj = 0; ifj < theBigTree.ak8jets_px->size(); ++ifj)
+			  TLorentzVector tlv_fj (theBigTree.ak8jets_px->at(ifj) , theBigTree.ak8jets_py->at(ifj) , theBigTree.ak8jets_pz->at(ifj) , theBigTree.ak8jets_e->at(ifj));
+			  if (tlv_fj.Pt() < 250) continue; 
```

Comment:
Can you refresh my memory about the need for this cut?

Reply:
The boosted regime is expected to be dominated for pT > 400 GeV so this is a very loose cut to ensure that there is some boost.